test/k8sT/istio.go

```diff
@@ -37,27 +37,27 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sIstioTest", func() {
 		// installed.
 		istioSystemNamespace = "istio-system"
 
-		istioVersion = "1.8.2"
+		istioVersion = "1.9.6"
 
 		// Modifiers for pre-release testing, normally empty
-		prerelease     = "" // "-beta.1"
-		istioctlParams = ""
+		prerelease     = "-beta.1"
+		istioctlParams = // ""
 
 		// Keeping these here in comments serve multiple purposes:
 		// - remind how to test with prerelease images in future
 		// - cause CI infra to prepull these images so that they do not
 		//   need to be pulled on demand during the test
-		// " --set values.pilot.image=docker.io/cilium/istio_pilot:1.8.2" +
-		// " --set values.global.proxy.image=docker.io/cilium/istio_proxy:1.8.2" +
-		// " --set values.global.proxy_init.image=docker.io/cilium/istio_proxy:1.8.2" +
-		// " --set values.global.proxy.logLevel=trace"
-		// " --set values.global.logging.level=debug"
+		" --set values.pilot.image=docker.io/jrajahalme/istio_pilot:1.9.6" + prerelease +
```

Comment:
Please update the note above, which tells that the lines are commented: `// Keeping these here in comments`...